src/types/webhooks.ts

```diff
@@ -0,0 +1,8 @@
+export type Webhook = {
+  uuid: string;
+  url: string;
+  headers?: Record<string, string>;
+  isEditable: boolean;
+};
+
+export type WebhookPayload = Omit<Webhook, "uuid" | "isEditable">;
```

Comment:
When doing a patch, the URL is not mandatory, so I guess we should do a third type dedicated to the `WebhookPatchPayload` where the URL is not mandatory?
From what I see in the doc, we cannot, right? https://www.typescriptlang.org/docs/handbook/utility-types.html#parameterstype
Maybe we have to do it the other way around and make the webhook URL always optional, and then make all the types we need with that? https://www.typescriptlang.org/docs/handbook/utility-types.html#nonnullabletype

I'm just proposing that because I'm learning at the same time but show me if there is a better way to do that 😁